src/java/htsjdk/samtools/util/SimpleInterval.java

```diff
@@ -0,0 +1,45 @@
+package htsjdk.samtools.util;
+
+
+/**
+ * minimal immutable class representing a 1-based closed ended genomic interval
+ */
+public class SimpleInterval implements Locatable {
+
+    private final int start;
+    private final int end;
+    private final String contig;
+
+    public SimpleInterval(final String contig, final int start, final int end){
+        if(contig == null){
+            throw new IllegalArgumentException("contig cannot be null");
```

Comment:
Can the current implementation of `SimpleInterval` represent an unmapped interval? If so, how?